index.html

```diff
@@ -1,29 +1,152 @@
 <!DOCTYPE html>
-<html lang="en-ca">
-  <head>
-    <!-- Set the page title -->
-    <title>Assignment 02</title>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, intial-scale=1.0">
+        <Title>House of Shelby</Title>
+        <link href="style.css"rel="stylesheet" type="text/css">
+        <link href="reset.css"rel="stylesheet" type="text/css">   
+    </head>
+    <body>
+        <style>
+            body{
+                background-color:#100D0B;
+            }
+            
+        </style>
```

Comment:
Try to keep all styles in CSS files. 

Inline and embedded CSS properties are like wearing socks with sandals. I mean, you can do it, but, we live in a civil society.